src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/DiaryDayViewController.swift

```diff
@@ -124,9 +122,19 @@ class DiaryDayViewController: UIViewController, UITableViewDataSource, UITableVi
 	override func viewWillAppear(_ animated: Bool) {
 		super.viewWillAppear(animated)
 		
-		// After change the tab bar items and switch back again to the Journal tab then need to set the same appearance as before.
-		setupTabBarAppearance()
-		
+		/*
+		 In iOS 15, the tab bar background automatically adjusts it self, when there is no content on the back it becomes transparent
+		 where as it has a background when there is content on the back. Unfortunately, in our case the tab bar remains transparent even
+		 through there is content on the back, so we fix this by overriding the appearance with a background.
+		 Solution is inspired from: https://developer.apple.com/forums/thread/682420
+		 */
```

Comment:
From this comment, it seems like this will only fix the issue under iOS 15, but I'm on iOS 16 and have the same bug. Will you fix that separately, or is the comment just outdated?

Reply:
It will fix for iOS15+.

Reply:
Thanks!